lib/graph/graph.js

```diff
@@ -43,7 +43,7 @@ const log = getLogger("generateProjectGraph");
 export async function graphFromPackageDependencies({
 	cwd, rootConfiguration, rootConfigPath,
 	versionOverride, cacheMode, resolveFrameworkDependencies = true,
-	workspaceName /* TODO 4.0: default workspaceName to "default" ? */,
+	workspaceName="default",
```

Comment:
There's no test assertion that makes sure "default" is used as default value. Renaming this string does not fail any test case in `test/lib/graph/graph.js`.

Reply:
Thanks! I have created a test for the "default" workspace name